hydra-plutus/src/Hydra/Contract/Head.hs

```diff
@@ -208,13 +209,15 @@ checkCollectCom context@ScriptContext{scriptContextTxInfo = txInfo} headContext
 
   (expectedChangeValue, collectedCommits, nTotalCommits) =
     traverseInputs
-      (negate (txInfoAdaFee txInfo), encodeBeginList, 0)
+      (negate (txInfoAdaFee txInfo), [], 0)
       (txInfoInputs txInfo)
 
   expectedOutputDatum :: Datum
   expectedOutputDatum =
-    let utxoHash =
-          (collectedCommits <> encodeBreak)
+    let encodedCommits =
```

Comment:
Add a mutation in `CollectCom` to change the `TxOutRef` so that the ordering becomes different: This will trigger this code and should fail the validator